kubectl-rook-ceph.sh

```diff
@@ -273,6 +276,97 @@ function run_purge_osd() {
     rook ceph osd remove --osd-ids=$1 --force-osd-removal=$force_removal"
 }
 
+####################################################################################################
+# 'kubectl rook-ceph debug commands
+####################################################################################################
+
+function update_deployment_spec_command(){
+  deployment_spec=$1
+  main_command='["sleep","infinity"]'
+  deployment_spec=$(jq .template.spec.containers[0].command="$main_command" <<< $deployment_spec)
+  deployment_spec=$(jq .template.spec.containers[0].args=[] <<< $deployment_spec)
+  echo "$deployment_spec" # for returning the updated deployment_spec
+}  
+
+function update_deployment_spec_image(){
+  deployment_spec=$1
+  alternalte_image=\"$2\"
+  deployment_spec=$(jq .template.spec.containers[0].image="$alternalte_image" <<< $deployment_spec)
+  echo "$deployment_spec" # for returning the updated deployment_spec
+}
+
+function verify_deployment_exist(){
+  if [[ $deployment_name != *"mon"* ]] && [[ $deployment_name != *"osd"* ]]; then
```

Comment:
```suggestion
  if [[ $deployment_name != "rook-ceph-mon-"* ]] && [[ $deployment_name != "rook-ceph-osd-"* ]]; then
```

This could still allow debugging on an RGW deployment where the CephObjectStore is named `pokemon` (rook-ceph-rgw-pokemon-a). It is best to make these checks as specific as possible so we don't allow unsupported cases.